utils/config_manager.py

```
@@ -174,9 +174,13 @@ def migrate_catgirl_reserved(catgirl_data: dict) -> bool:
     model_type = str(
         get_reserved(catgirl_data, "avatar", "model_type", default="", legacy_keys=("model_type",))
     ).strip().lower()
-    if model_type not in {"live2d", "vrm"}:
+    if model_type not in {"live2d", "vrm", "live3d"}:
         has_vrm = catgirl_data.get("vrm") or get_reserved(catgirl_data, "avatar", "vrm", "model_path")
-        model_type = "vrm" if has_vrm else "live2d"
+        has_mmd = catgirl_data.get("mmd") or get_reserved(catgirl_data, "avatar", "mmd", "model_path")
+        model_type = "live3d" if (has_vrm or has_mmd) else "live2d"
+    # 归一化：旧配置中的 'vrm' 统一为 'live3d'
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)